core/includes/form.inc

```diff
@@ -4750,6 +4756,7 @@ function theme_textarea($variables) {
 function theme_password($variables) {
   $element = $variables['element'];
   $element['#attributes']['type'] = 'password';
+  $element['#attributes']['spellcheck'] = 'false';
```

Comment:
@yorkshire-pudding I really prefer the second way I suggested, with `=+`. The problem of doing it this way is that you will force false on spellcheck, regardless of what the form definition has. I realize this is really an edge case, but the way element definition work is that the preprocess and theming functions always respect the original definition of the element in the form function. Right now this code doesn't.

For example, if I do (I know this is a crazy edge case):

```php
function testing_stuff_test_form($form, &$form_state) {
  $form['password'] = array(
    '#type' => 'password',
    '#title' => 'Password',
    '#maxlength' => 64,
    '#size' => 15,
    '#disabled' => TRUE,
    '#spellcheck' => 'true',
  );

  return $form;
}
```

I expressly set `#spellcheck` to 'true'. Yet the theming function ignores this and forces `false` on me. 

Again, I know this is an edge case, but for the sake of consistency, we should code theming and preprocess functions to respect the original definition of the element.

Reply:
@yorkshire-pudding forget it - I'm wrong! spellcheck can't be configured in the form definition. Resolving this comment now...

Reply:
Actually only you can resolve it - can't do that on my end.

Reply:
Not sure why this was posted twice. The above is incorrect.